python-sdk/tutorials/automl-with-azureml/image-object-detection/coco2jsonl.py

```diff
@@ -61,7 +61,8 @@ def _populate_label(self, annotation):
         self.json_lines_data[index]["label"].append(label)
 
     def _populate_isCrowd(self, label, annotation):
-        label["isCrowd"] = annotation["iscrowd"]
+        if 'iscrowd' in annotation.keys():
```

Comment:
minor: I think it works without ".keys()"

Reply:
yes, both works fine.